src/FAudio.c

```diff
@@ -240,6 +240,185 @@ void FAudio_UnregisterForCallbacks(
 	LOG_API_EXIT(audio)
 }
 
+#ifdef FAUDIO_DUMP_VOICES
+static FAudioIOStreamOut *DumpWMA_write_RIFF_openRWops(
+	const FAudioSourceVoice *voice,
+	const FAudioWaveFormatEx *format,
+	const char *mode
+) {
+	char loc[64];
+	uint16_t format_tag = format->wFormatTag;
+	uint16_t format_ex_tag = 0;
+	if (format->wFormatTag == FAUDIO_FORMAT_EXTENSIBLE)
+	{
+		/* get the GUID of the extended subformat */
+		const FAudioWaveFormatExtensible *format_ex =
+				(const FAudioWaveFormatExtensible*) format;
+		format_ex_tag = (uint16_t) (format_ex->SubFormat.Data1);
+	}
+	if (format->wFormatTag == FAUDIO_FORMAT_WMAUDIO2)
+	{
+		format_tag = FAUDIO_FORMAT_EXTENSIBLE;
+		format_ex_tag = FAUDIO_FORMAT_WMAUDIO2;
+	}
+	SDL_snprintf(
```

Comment:
`FAudio_snprintf` is available to use in place of direct SDL_snprintf.

Reply:
replaced, good catch